privacyidea/api/container.py

```diff
@@ -140,7 +141,7 @@ def remove_token(container_serial):
     token = get_one_token(serial=serial)
     res = False
     if token:
-        container.remove_token(token)
+        count = container.remove_token(token.get_serial())
```

Comment:
Unused variable `count` could be removed.

Reply:
683a31b